entities/vectorindex/hnsw/config.go

```diff
@@ -58,6 +58,7 @@ type UserConfig struct {
 	BQ                     BQConfig `json:"bq"`
 	SQ                     SQConfig `json:"sq"`
 	FilterStrategy         string   `json:"filterStrategy"`
+	Multivector            bool     `json:"multivector"`
```

Comment:
Do you think we should capitalize everywhere as `multiVector` / `MultiVector`? Will we need more parameters later on?

Reply:
At the moment `multivector` is just a flag, but in the future can change. For example if we introduce ColBERTv2 which uses the centroids, in that case it could be a parameter.